core/src/test/java/io/questdb/test/cutlass/http/ScoreboardHandlingTest.java

```diff
@@ -0,0 +1,174 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2024 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.test.cutlass.http;
+
+import io.questdb.PropertyKey;
+import io.questdb.ServerMain;
+import io.questdb.std.Files;
+import io.questdb.std.str.Path;
+import io.questdb.test.AbstractBootstrapTest;
+import io.questdb.test.tools.TestUtils;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static io.questdb.test.tools.TestUtils.*;
+
+public class ScoreboardHandlingTest extends AbstractBootstrapTest {
+
+    static String backupRoot;
+
+    @BeforeClass
+    public static void setUpStatic() throws Exception {
+        AbstractBootstrapTest.setUpStatic();
+        unchecked(() -> {
+            backupRoot = temp.newFolder("backups").getAbsolutePath();
+        });
+    }
+
+    @Test
+    public void testPermissionsAreUpdatedWhenColumnIsRenamed() throws Exception {
```

Comment:
nit: the test name is weird considering that OSS doesn't have ACL.

Reply:
this is fixed